_docs/tasks/security/role-based-access-control.md

```diff
@@ -67,9 +67,11 @@ Run the following command to enable Istio RBAC for "default" namespace.
 
 
 ```bash
-kubectl apply -f samples/bookinfo/kube/istio-rbac-enable.yaml
+istioctl create -f samples/bookinfo/kube/istio-rbac-enable.yaml
```

Comment:
I think we can also mention that the user could use 'make istioctl-install' to install the istioctl if they haven't done so.
And if the user don't want to use istioctl, they can still use kubectl but just doesn't have the validation part.

Reply:
istioctl is automatically installed when a user installs Istio.